playbooks/nvidia-driver.yml

```diff
@@ -1,6 +1,15 @@
+---
 - hosts: all
   become: true
   tasks:
+    - name: run setup to detect OS
+      setup:
+
+    - name: install lspci
+      package:
```

Comment:
I think you need an `update_cache`, when I installed CentOS 7 via MAAS it hadn't been updated.

Reply:
Ah. Ok. My test setup ran it at some point, I guess.

Reply:
Fixed. I couldn't decide whether I wanted to have two `update_cache` blocks and a `package` block or a `yum` and an `apt` 🤷‍♂️